### ISSUE TITLE
Unable to load geometry into vis-gui.py demo

### ISSUE THREAD

**Issue by cygann** [intel-isl/Open3D#2268]
**IMPORTANT: Please use the following template to report the bug.**
_____

**Describe the bug**
I am interested in using the new gui module for 3d point cloud visualizations with a UI, however, I cannot get the vis-gui.py demo to successfully display a point cloud. Instead, I get this error after successfully reading the file:

```
FEngine (64 bits) created at 0x7f8a70a67000 (threading is enabled)
FEngine resolved backend: OpenGL
[Info] /Users/nataliecygan/Desktop/redleader/point_cloud_viz/inputs/pointcloud.pcd appears to be a point cloud
[Info] Successfully read /Users/nataliecygan/Desktop/redleader/point_cloud_viz/inputs/pointcloud.pcd
Traceback (most recent call last):
  File "vis-gui.py", line 759, in <module>
    main()
  File "vis-gui.py", line 749, in main
    w.load(path)
  File "vis-gui.py", line 724, in load
    self._scene.scene.add_geometry(geometry, self.settings.material)
TypeError: add_geometry(): incompatible function arguments. The following argument types are supported:
    1. (self: open3d.pybind.visualization.rendering.Open3DScene, name: str, geometry: open3d.pybind.geometry.Geometry3D, material: open3d.pybind.visualization.rendering.Material, add_downsampled_copy_for_fast_rendering: bool = True) -> None
```

I was wondering if I could get some pointers on this. Thanks!

**To Reproduce**

I built open3d from source and tried to run the [vis-gui.py example](https://github.com/intel-isl/Open3D/blob/master/examples/python/GUI/vis-gui.py). I made no changes to the file before running it.

**Expected behavior**
Expected to add the specified point cloud file to the visualization. 

**Screenshots**
![Screen Shot 2020-09-01 at 4 26 20 PM](https://user-images.githubusercontent.com/16779401/91916984-7ebe0d00-ec73-11ea-90f1-42d9c0bae735.png)

**Environment (please complete the following information):**

- Operating system: OSX 10.15.6
- Python version: Python 3.7.1
- Open3D version: From source
- Is this remote workstation?: no
- How did you install Open3D?: build from source
- Compiler version (if built from source): Apple clang 11.0.3

**Comment by germanros1987** [comment]
Hi @cygann, 

after building master, did you also build the pip package and install it? Just double-checking.

**Comment by cygann** [comment]
Yes, I did. I'm able to play around with the other features in the demo, but it seems to just be adding geometry that won't work for me.

**Comment by germanros1987** [comment]
I can reproduce the issue. We are looking into this to come up with a hotfix.

**Comment by cygann** [comment]
Thanks so much! For future reference, is there documentation available for working with the gui module in python?

**Comment by germanros1987** [comment]
The problem has been solved by #2273 , included in the latest version of master. Please, give it a try.

Regarding the documentation, you are testing features that have not been released yet. We will make the documentation available at release time.
